src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -249,6 +250,10 @@ public function addResource(ResourceInterface $resource)
             return $this;
         }
 
+        if ($resource instanceof GlobResource && $this->inVendors($resource->getPrefix())) {
```

Comment:
Constructing a GlobResource is done after some non trivial logic in FileLoader so I'm not sure we want to expose helper for that to me. This special handling is fine to me (no BC break by definition, and aligned with the purpose of resource tracking for the container).